Gemfile

```diff
@@ -71,6 +71,7 @@ group :development, :test do
   gem 'rspec-collection_matchers'
   gem 'webrick',        '~> 1.3'
   gem 'parallel_tests'
+  gem 'literate_randomizer'
```

Comment:
Faker Lorem module can be used for this:

http://www.rubydoc.info/github/stympy/faker/Faker/Lorem